source/configuration/read.txt

```diff
@@ -282,8 +267,8 @@ This partitioner is not compatible with hashed shard keys.
      - Description
 
    * - ``partitioner.options.partitionKey``
-     - A comma-delimited list of fields by which to split the 
-       collection data. The fields should be indexed and contain unique 
+     - A comma-delimited list of fields by which to split the
+       collection data. The fields should be indexed and contain unique
        values.
 
        **Default:** ``_id``
```

Comment:
I created a new sub-section in Partitioners for this since the list of partitioners within the config table was replaced as part of the ticket. Does this partitioner have any options?

Reply:
No options - its just the simplest partitioner and will work with any users aggregation pipelines eg `$group`